internal/unix-pipeline.go

```diff
@@ -0,0 +1,122 @@
+package internal
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+
+	app "github.com/dilyara4949/unix-pipeline"
+)
+
+const (
+	cat            app.Operation = "cat"
+	grep           app.Operation = "grep"
+	ssort          app.Operation = "sort"
```

Comment:
If name for variable same as name for import, you can just add alias for import `s "sort"`